src/fragments/lib/graphqlapi/js/authz.mdx

```diff
@@ -16,15 +16,38 @@ In order to use this feature with the Amplify GraphQL Client the `API.graphql({.
 
 This is an example of using `AWS_IAM` as an authorization mode:
 
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { GraphQLQuery, GRAPHQL_AUTH_MODE } from '@aws-amplify/api';
+import { CreateTodoMutation } from './API';
+
+// Creating a post is restricted to IAM 
+const createdTodo = await API.graphql<GraphQLQuery<CreateTodoMutation>>({
+  query: queries.createTodo,
+  variables: { input: todoDetails },
+  authMode: GRAPHQL_AUTH_MODE.AWS_IAM
+});
+```
+
+</Block>
+<Block name="JavaScript">
+
 ```js
+import { GRAPHQL_AUTH_MODE } from '@aws-amplify/api';
```

Comment:
isn't this missing the "API" import?

Reply:
It's supposed to be more of a diff from the last example... But if you think it makes things clearer, I can add all the imports used by the snippet